src/Runner.php

```diff
@@ -99,7 +99,7 @@ class Runner
      *
      * @since [*next-version*]
      *
-     * @param string $instanceId The prefix to use for options in the wp_options table.
+     * @param int    $instanceId The prefix to use for options in the wp_options table.
```

Comment:
This is also intended to be used as a string. See my other comment and line 247 in this file.

Reply:
Done.